pontoon/base/signals.py

```diff
@@ -156,3 +157,15 @@ def assign_project_locale_group_permissions(sender, **kwargs):
 def create_user_profile(sender, instance, created, **kwargs):
     if created:
         UserProfile.objects.create(user=instance)
+
+
+@receiver(pre_save, sender=Translation)
+def update_entity_document(sender, instance, raw,  **kwargs):
+    """
+    Update entity document for the faster search queries.
+    """
+    # Don't update the document during the loaddata operation.
+    if raw:
+        return
+
+    instance.entity_document = instance.entity.search_document
```

Comment:
Yes, they do. I executed migrations and sync_projects and entity_document was populated.